NitroxModel/Packets/EntityDestroy.cs

```diff
@@ -6,7 +6,9 @@ namespace NitroxModel.Packets;
 [Serializable]
 public class EntityDestroy : Packet
 {
-    public NitroxId EntityId { get; }
+    public NitroxId EntityId { get; set; }
+
+    public EntityDestroy() { }
```

Comment:
Is there a reason this empty constructor is needed?

Reply:
No, the revert must have missed this file.